install_sct

```diff
@@ -591,7 +591,7 @@ osx)
 esac
 
 # Run conda installer
-run bash "$TMP_DIR/miniconda.sh" -p "$SCT_DIR/$PYTHON_DIR" -b -f
+run bash "$TMP_DIR/miniconda.sh" -p "./$PYTHON_DIR" -b -f
 
 # create conda environment
 python/bin/conda create -y -n venv_sct python=3.7.11
```

Comment:
Yeah, #3564 looks like it will get in the way of debugging Maxime's issue, because the `symbolic links` error happens prior to the rest of the install. 

I'd maybe try pinning `ncurses=6.2` when making the conda environment (at least temporarily)? 

```suggestion
python/bin/conda create -y -n venv_sct python=3.7.11 ncurses=6.2
```